tests/Yaapii.Atoms.Tests/Text/ComparableTextTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Yaapii.Atoms.Text.Tests
+{
+    public sealed class ComparableTextTests
+    {
+		[Fact]
+		public void Compares()
+		{
+			Assert.True(
+				new ComparableText(
+					new TextOf("Hallo Welt")
+				).CompareTo(
+					new TextOf("Tschüss Welt")
+				) <= 0 
+			);			
+		}
+
+		[Fact]
+		public void Equal()
```

Comment:
Name like what you expect as behaviour from the object, e.g.: SeesDIfferences or similar.